base_eeg/test.py

```diff
@@ -1,32 +1,30 @@
 import mne
 import pathlib
-import json
-import autoreject as ar
-from find_montage import get_montage
 from mne_bids import BIDSPath, read_raw_bids
 import finalRej as fr
-    
+
+
 def main():
```

Comment:
i think the linter is complaining about only leaving 1 space here , I think you you put the comment one line lower it the complaint will go away. documentation [here](https://www.flake8rules.com/rules/E305.html)